FeedAPIChallenge/Feed Feature/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Yonic Surny on 11/07/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImagesMapper {
```

Comment:
The convention in this codebase is to omit the default access level keyword `internal`. Please remove it.
```suggestion
final class FeedImagesMapper {
```